packages/dx-react-grid/docs/reference/data-type-provider.md

```diff
@@ -57,5 +58,6 @@ none
 
 Name | Plugin | Type | Description
 -----|--------|------|------------
+availableFilterOperations | [Getter](../../../dx-react-core/docs/reference/getter.md) | { columnName: Array&lt;string&gt; } | Provides information about columns with customized filter operations.
```

Comment:
Why this getter is not a function like other getters. For example, `isColumnSortingEnabled`.